azure-iot-device/azure/iot/device/common/mqtt_protocol_wrapper.py

```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 
-class MQTTProvider(object):
+class MQTTProtocolWrapper(object):
```

Comment:
MQTTClientWrapper maybe? If Paho is the MQTT Client, and we're wrapping it, that seems the most appropriate thing.

Reply:
Interface is probably not a good word. Even the class name MQTTProtocolWrapper or MQTTClientWrapper sounds good. What about MQTTClientOperator ? or MQTTOperator ? 

---
In reply to: [295558178](https://github.com/Azure/azure-iot-sdk-python-preview/pull/111#discussion_r295558178) [](ancestors = 295558178)